crawl-ref/source/dat/descript/de/gods.txt

```diff
@@ -121,16 +121,6 @@ verzehren, die von ihren Anhängern woanders im Verlies liegen gelassen wurden.
 Besonders strenggläubige Anhänger werden genauso formlos wie ihr Gott, weil
 ihre Eigenschaften und Mutationen sich unter Jiyvas Einfluss verändern.
 %%%%
-Zin powers
```

Comment:
I don't know much about how crawl translations work but it seems a shame to delete all these because one word is wrong, maybe being the translational equivalent of protected isn't so far off the new version?